src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6368,6 +6368,46 @@ IArrayInitializerOperation popAndAssembleArrayInitializerValues(IArrayInitialize
             }
         }
 
+        public override IOperation? VisitCollectionExpression(ICollectionExpressionOperation operation, int? argument)
```

Comment:
Looking at `VisitInterpolatedStringHandlerCreation` below, I would have expected the CFG for collection expression to surface the nitty-gritty details of the collection evaluation. It sounds like this was discussed/approved in API review, but I'm not clear on whether this is the level of abstraction we want moving forward?

Reply:
Resolved offline. This is the result of a case-by-case discussion in API review considering the needs of the CFG customer. Thanks